news/2021-01-14-performance-points-updates.md

```diff
@@ -0,0 +1,127 @@
+
+For those unfamiliar with the technical minutiae of the system, please consult [the wiki article](/wiki/Performance_points) for an overview of the general ideas involved. 
+
+To summarize the article very briefly (you should really read the whole thing on your own first!), pp encompasses several core values as follows:
+
+- **Aim**, which considers how difficult it is to consistently hit notes back-to-back during play (affected by things like approach rate, jumps, Flashlight, Hidden, Hard Rock and more)
+- **Speed**, which considers how quickly new elements are presented to the player during the course of the map (affected by things like the number of hit objects over time, BPM, Double Time and Half Time)
+- **Accuracy**, which considers the player's ability to stay on time rhythmically and how difficult this is to achieve (affected by things like Hidden, Hard Rock, Flashlight and more)
+- **Strain**, which considers how often and how long a player has to endure intense or technically demanding patterns during play (affected by things like streams, fast jumps and more)
+
+**The changes listed below apply solely to the osu! game mode.**
+
+Let's get started with the details:
+
+## Approach Rate 11 Changes 
+
+Previously, any map with an approach rate (hereafter referred to as AR) higher than 10.33 was affected by a buff that increased the value of the speed and aim components by a flat 20%.
+
+Maps at or above AR 11 are often considered to be similar to the Flashlight mod in that they become considerably more difficult as a map progresses. This fact was not reflected in the previous version of the algorithm with its static 20% increase.
+
+As suggested by [**Xexxar**](https://osu.ppy.sh/users/2773526), the following change has been applied:
+
+- The buff for AR 11 maps now scales from 0% at 0 objects, to 26.66% at a maximum of 1000 objects. The previous 20% buff is now attained at around 750 objects instead.
+    - Old value: 20% flat.
+
+In essence, this nerfs the amount of pp awarded for shorter AR 11 maps, but buffs the amount gained for longer, more difficult performances.
+
+Consult the graph below for a visualization of the changes:
+
+![](/wiki/shared/news/2021-01-14-performance-points-updates/ar_multiplier.png)
+
+## No Fail Changes
+
+Before today's changes, the performance points algorithm applied a flat 10% reduction in the maximum possible pp (not score) you could receive for a play made with the No Fail mod active.
+
+As suggested by [**StanR**](https://osu.ppy.sh/users/7217455) in [this pull request](https://github.com/ppy/osu-performance/pull/127), the following change has been applied:
+
+- No Fail's pp multiplier now scales based on the amount of misses.
+    - New value: 1.0x base, with -0.02x applied for each miss to a maximum reduction back to 0.9x
+    - Old value: 0.9x (always)
```

Comment:
```suggestion
    - **New value**: 1.0x base, with -0.02x applied for each miss to a maximum reduction back to 0.9x
    - **Old value**: 0.9x (always)
```
kinda felt like these should be formatted in bold. please apply the same to the rest if you agree